test/resources.go

```diff
@@ -45,10 +45,10 @@ func ToUnstructured(t *testing.T, in interface{}) *unstructured.Unstructured {
 // expected to be able to interact with.
 func AddTektonResources(clientset *fakekubeclientset.Clientset) {
 	nameKind := map[string]string{
-		"triggertemplates":  "TriggerTemplate",
 		"pipelineruns":      "PipelineRun",
 		"taskruns":          "TaskRun",
 		"pipelineresources": "PipelineResource",
+		"triggertemplates":  "TriggerTemplate",
```

Comment:
any reason to change the position of these attributes

Reply:
I removed it thinking it was not necessary but then added it back after tests started failing. Will revert it.